src/docs/product/integrations/index.mdx

```diff
@@ -23,13 +23,14 @@ For more details, see the [full Integration Platform documentation](/product/int
 
 | Integration                                       | Issue alerts | Metric alerts | Link Unfurling |
 | ------------------------------------------------- | ------------ | ------------- | -------------- |
-| [Amixr](/product/integrations/amixr/)             | X            |               |                |
+| [Amixr](/product/integrations/notification-incidents/amixr/)             | X            |               |                |
 | Datadog                                           | X            |               |                |
+| [Microsoft Teams](/product/integrations/notification-incidents/msteams/) | X            | X             |                |
 | OpsGenie                                          | X            |               |                |
-| [PagerDuty](/product/integrations/pagerduty/)     | X            | X             |                |
+| [PagerDuty](/product/integrations/notification-incidents/pagerduty/)     | X            | X             |                |
 | Pushover                                          | X            |               |                |
-| [Slack](/product/integrations/slack/)             | X            | X             | X              |
-| [Microsoft Teams](/product/integrations/msteams/) | X            | X             |                |
+| [Slack](/product/integrations/notification-incidents/slack/)             | X            | X             | X              |
+| [Spike.sh](/product/integrations/notification-incidents/slack/)          |              |               |                |
```

Comment:
@manuzope - I'm adding Spike.sh to the table on the main Integrations page. Is this the right table for it to be in on that page (Notification & Incidents) and if so, which features does it have in terms of the table columns?

Reply:
Ah yes, thank you for doing this. This is the correct table. It has issue alerts and metric alerts, not link unfurling.

Reply:
Thank you!!